ops/prod/docker-compose.yml

```
@@ -0,0 +1,71 @@
+    restart: unless-stopped
+    ports: ["80:80","443:443"]
+    volumes:
+      - ../data/caddy:/data
+      - ./Caddyfile:/etc/caddy/Caddyfile:ro
+    depends_on: [frontend]
+
+  frontend:
+    build:
+      context: .
+      dockerfile: ./ops/prod/Dockerfile.frontend
+    env_file: [ ./frontend.env ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)